feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -469,6 +483,8 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 		dutConfPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
 		nbrList := buildNbrList(ateAS)
 		dutConf := bgpWithNbr(dutAS, nbrList)
+		rpl := configureRoutingPolicy(&oc.Root{})
+		gnmi.Replace(t, dut, gnmi.OC().RoutingPolicy().Config(), rpl)
```

Comment:
This RPL config replace is deleting the previous config done at https://github.com/openconfig/featureprofiles/blob/main/feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go#L462.

Could you change the rplPermitAll name string to "ALLOW", so that it's the same

Reply:
I actually haven't noticed this one behind a deviation, maybe we can keep the rpl configured by default if it's set under the peer-group instead?

Reply:
Yes, route-policy can be configured by default , but the RPL name needs to match with peer-group config referenced below https://github.com/openconfig/featureprofiles/blob/main/feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go#L194

Reply:
Sure I'll refactor and commit either later today or earlier tomorrow!

Reply:
Also existing function configureRoutePolicy can be used to configure RPL, to avoid duplication.

Reply:
Streamlined suggestions in latest commit.

Reply:
Thanks, its good for us now